Index: openmoko/trunk/src/host/qemu-neo1973/openmoko/download.sh
===================================================================
--- openmoko.orig/trunk/src/host/qemu-neo1973/openmoko/download.sh
+++ openmoko/trunk/src/host/qemu-neo1973/openmoko/download.sh
@@ -52,7 +52,7 @@ sleep 2
 download () {
 	[ -s "$1" ] && return
 	rm -rf "$1"
-	wget "$download_dir/$1"
+	wget -c "$download_dir/$1"
 }
 
 download "$kernel_image" || exit -1
