---
 trunk/src/host/qemu-neo1973/vl.h |    1 +
 1 file changed, 1 insertion(+)

Index: openmoko/trunk/src/host/qemu-neo1973/vl.h
===================================================================
--- openmoko.orig/trunk/src/host/qemu-neo1973/vl.h
+++ openmoko/trunk/src/host/qemu-neo1973/vl.h
@@ -293,6 +293,7 @@
 #define CHR_IOCTL_PP_READ_STATUS      7
 
 #define CHR_IOCTL_MODEM_HANDSHAKE     8
+#define CHR_IOCTL_MODEM_SET_AFC       9
 
 typedef void IOEventHandler(void *opaque, int event);
 
