---
 trunk/oe/conf/distro/openmoko.conf |    4 ++++
 1 file changed, 4 insertions(+)

Index: openmoko/trunk/oe/conf/distro/openmoko.conf
===================================================================
--- openmoko.orig/trunk/oe/conf/distro/openmoko.conf
+++ openmoko/trunk/oe/conf/distro/openmoko.conf
@@ -26,3 +26,7 @@
 PREFERRED_VERSION_glibc-intermediate ?= "2.4"
 
 PREFERRED_VERSION_lmsensors-apps ?= "2.10.1"
+
+CVS_TARBALL_STASH = "\
+		     http://downloads.openmoko.org/sources/ \
+		    "
