diff -ur openembedded.orig/packages/linux-libc-headers/linux-libc-headers_2.6.18.bb openembedded/packages/linux-libc-headers/linux-libc-headers_2.6.18.bb
--- openembedded.orig/packages/linux-libc-headers/linux-libc-headers_2.6.18.bb	2007-07-21 03:09:55.000000000 -0300
+++ openembedded/packages/linux-libc-headers/linux-libc-headers_2.6.18.bb	2007-07-21 18:06:01.000000000 -0300
@@ -52,7 +52,7 @@
 	rm -rf ${STAGE_TEMP}
 	mkdir -p ${STAGE_TEMP}
 	oe_runmake headers_install INSTALL_HDR_PATH=${STAGE_TEMP}/usr ARCH=${ARCH}
-	if [ "${ARCH}" == "arm" ]; then
+	if [ "${ARCH}" = "arm" ]; then
 		cp include/asm-arm/procinfo.h ${STAGE_TEMP}${includedir}/asm
 	fi
 	install -d ${STAGING_INCDIR}
