Index: openmoko/trunk/oe/packages/openmoko-apps/openmoko-simplemediaplayer_svn.bb
===================================================================
--- openmoko/trunk/oe/packages/openmoko-apps/openmoko-simplemediaplayer_svn.bb	(revision 1004)
+++ openmoko/trunk/oe/packages/openmoko-apps/openmoko-simplemediaplayer_svn.bb	(working copy)
@@ -5,3 +5,11 @@
 
 inherit openmoko
 
+do_install_prepend() {
+	touch mkinstalldirs
+}
+
+FILES_${PN} += " \
+                ${datadir}/images \
+		${libdir}/bmp/*/*.so \
+	       "
