---
 trunk/oe/classes/base.bbclass |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: openmoko/trunk/oe/classes/base.bbclass
===================================================================
--- openmoko.orig/trunk/oe/classes/base.bbclass
+++ openmoko/trunk/oe/classes/base.bbclass
@@ -717,7 +717,7 @@
         bb.error( "Revision fetching failed: '%s', falling back to current date." % value)
         return FALLBACKREVS
 
-    bb.note("Retrieved remote revisions: %r" % revs)
+    # bb.note("Retrieved remote revisions: %r" % revs)
     
     return revs
 
