---
 trunk/oe/conf/site.conf |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: openmoko/trunk/oe/conf/site.conf
===================================================================
--- openmoko.orig/trunk/oe/conf/site.conf
+++ openmoko/trunk/oe/conf/site.conf
@@ -21,4 +21,4 @@
 SOURCEFORGE_MIRROR = "http://heanet.dl.sf.net"
 PARALLEL_MAKE = ""
 
-SRCDATE = "20060814"
+# SRCDATE = "20060814"
